.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Install python dependencies
         run: |
-          pip install molecule[docker] yamllint ansible-lint docker openwisp-utils[qa]
+          pip install molecule[docker,ansible] yamllint ansible-lint docker openwisp-utils[qa]
```

Comment:
I am noot entirely sure that we need this.

Reply:
Previously, the build was failing because of this, this was the error I saw:
```
Unable to find a working copy of ansible executable. Read https://molecule.readthedocs.io/en/latest/installation.htmlCompletedProcess(args='ansible --version', returncode=1, stdout='\n', stderr='Traceback (most recent call last):\n  File "/usr/bin/ansible", line 34, in <module>\n    from ansible import context\nModuleNotFoundError: No module named \'ansible\'\n')
```
I saw this error in both my local env and in the github actions CI. Molecule docs [suggested](https://molecule.readthedocs.io/en/latest/installation.html#pip) to add this.

Reply:
Ok, thanks for the explanation @Purhan :+1: